pkg/web/static/openapi.json

```diff
@@ -1275,551 +1292,824 @@
           }
         }
       },
-      "Pending": {
+      "APIAuthentication": {
         "type": "object",
+        "description": "The request object for the authentication endpoint.",
         "properties": {
-          "envelope_id": {
-            "type": "string"
-          },
-          "received_by": {
-            "type": "string"
+          "client_id": {
+            "type": "string",
+            "description": "The client ID for authentication."
           },
-          "received_at": {
+          "client_secret": {
+            "type": "string",
+            "description": "The client secret for authentication."
+          }
+        }
+      },
+      "LoginReply": {
+        "type": "object",
+        "description": "The response object for the authentication endpoint.",
+        "properties": {
+          "access_token": {
             "type": "string",
-            "format": "date-time"
+            "description": "The access token for the authenticated session."
           },
-          "message": {
-            "type": "string"
+          "token_type": {
+            "type": "string",
+            "description": "The type of the access token."
           },
-          "reply_not_after": {
+          "refresh_token": {
             "type": "string",
-            "format": "date-time"
+            "description": "The refresh token for obtaining a new access token."
           },
-          "reply_not_before": {
+          "expires_in": {
+            "type": "integer",
+            "description": "The duration in seconds until the access token expires."
+          }
+        }
+      },
+      "ReauthenticateRequest": {
+        "type": "object",
+        "description": "The request object for the reauthentication endpoint.",
+        "properties": {
+          "refresh_token": {
             "type": "string",
-            "format": "date-time"
+            "description": "The refresh token for reauthentication."
+          }
+        }
+      },
+      "PageQuery": {
+        "type": "object",
+        "description": "The request object for paginated endpoints.",
+        "properties": {
+          "page_size": {
+            "type": "integer",
+            "format": "int32",
+            "example": 50,
+            "default": 100,
+            "description": "The number of items to return per page."
           },
-          "extra_json": {
-            "type": "string"
+          "next_page_token": {
+            "type": "string",
+            "format": "base64",
+            "description": "The token for retrieving the next page of results."
           },
-          "transaction": {
-            "$ref": "#/components/schemas/Transaction"
+          "prev_page_token": {
+            "type": "string",
+            "format": "base64",
+            "description": "The token for retrieving the previous page of results."
           }
         }
       },
-      "SecureEnvelope": {
+      "IVMS101Certify": {
```

Comment:
Could you provide more detail about the inclusion of this object? I didn't see it used elsewhere.

Reply:
Probably a leftover from previous refactor steps I've done. Thank you for the catch.